rali/rali_pybind/example/tf_mnistTrainingExample/tf_mnist_classification_rali.py

```diff
@@ -0,0 +1,192 @@
+	tfrecord_features = tf.parse_single_example(tfrecord_serialized, features={
+		'image/height': tf.FixedLenFeature([], tf.int64),
+		'image/width': tf.FixedLenFeature([], tf.int64),
+		'image/class/label': tf.FixedLenFeature([], tf.int64),
+		'image/raw': tf.FixedLenFeature([], tf.string),
+	}, name='features')
+	image = tf.decode_raw(tfrecord_features['image/raw'], tf.float32)
+	image.set_shape([784])
+	label = tf.cast(tfrecord_features['image/class/label'], tf.int32)
+#	image_batch, label_batch = tf.train.batch([image, label], batch_size=bs)
+	return image, label
+
+#RALI pipeline
+class HybridPipe(Pipeline):
+	def __init__(self, feature_key_map, tfrecordreader_type, batch_size, num_threads, device_id, data_dir, crop, rali_cpu = True):
+		super(HybridPipe, self).__init__(batch_size, num_threads, device_id, seed=12 + device_id,rali_cpu=rali_cpu)
+		self.input = ops.TFRecordReader(path=data_dir, index_path = "", reader_type=tfrecordreader_type, user_feature_key_map=feature_key_map,
+										features={
+											'image/encoded':tf.FixedLenFeature((), tf.string, ""),
+											'image/class/label':tf.FixedLenFeature([1], tf.int64,  -1),
+											'image/filename':tf.FixedLenFeature((), tf.string, "")
+										},
+										)
+		rali_device = 'cpu' if rali_cpu else 'gpu'
+		decoder_device = 'cpu' if rali_cpu else 'mixed'
+		self.decode = ops.ImageDecoderRaw(user_feature_key_map=feature_key_map, device=decoder_device, output_type=types.RGB)
+		#self.res = ops.Resize(device=rali_device, resize_x=crop[0], resize_y=crop[1])
+		self.cmnp = ops.CropMirrorNormalize(device="cpu",
+											output_dtype=types.FLOAT,
+											output_layout=types.NCHW,
+											crop=(crop,crop),
+											image_type=types.GRAY,
+											mean=[0 ,0,0],
+											std=[255,255,255], mirror=0)
+		self.coin = ops.CoinFlip(probability=0.5)
+		print('rali "{0}" variant'.format(rali_device))
+
+	def define_graph(self):
+		inputs = self.input(name ="Reader")
+		images = inputs["image/encoded"]
+		labels = inputs["image/class/label"]
+		images = self.decode(images)
+		#rng = self.coin()
+		output = self.cmnp(images)
+		return [output, labels]
+
+# compute accuracy
+def compute_accuracy(predictions, labels):
+	correct_predictions = tf.equal(tf.argmax(predictions, 1), tf.argmax(labels, 1))
+	accuracy = tf.reduce_mean(tf.cast(correct_predictions, tf.float32))
+	return accuracy
+
+def train_mnist_rali(data_path, _rali_cpu, batch_size):
+
+	# setup keep_prob
+	#keep_prob = tf.placeholder(tf.float32)
+	input_X = tf.placeholder('float32',shape = (batch_size,784))
+	labels = tf.placeholder('float32',shape = (batch_size,10))
+	logits = neural_net(input_X)
+	cost = tf.reduce_mean( tf.nn.softmax_cross_entropy_with_logits(labels=labels, logits=logits), name="loss" )
+	optimizer = tf.train.AdamOptimizer().minimize(cost)
+	train_prediction = tf.nn.softmax(logits)
+	accuracy = compute_accuracy(train_prediction, labels)
+	correct_label = tf.argmax(labels, 1)
```

Comment:
Codacy found an issue: [local variable 'correct_label' is assigned to but never used (F841)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6366586)